marine/marine.py

```diff
@@ -10,10 +10,16 @@ class MarineResult(Structure):
 
 
 MARINE_RESULT_POINTER = POINTER(MarineResult)
+MARINE_PREF_T = c_char * 1024
```

Comment:
Expose it in `marine.h` and then get it from `libmarine.so`.

Reply:
I'm not a big fan of this approach. Why not have the API receive a simple string and convert it to `marine_pref_t` inside the C code?

Reply:
i thought about this, i just didn't want to add those malloc, strcpy and free in the `C` code